### ISSUE TITLE
Other architectures

### ISSUE THREAD

**Issue by japita-se** [ildoonet/tf-lcnn#4]
Theoretically, It would feasible to use the same concepts for more complex architecyures like tacotron or deepspeech?

**Comment by ildoonet** [comment]
Yes, sure. But you will need to tune hyperparameters for each new architecture.

**Comment by japita-se** [comment]
It would be great if someone would join the adventure :) my goal is to run deepspeech on a constrained board. I would try XNOR-net, but it seems that lcnn is superior.
